lib/gds_api/json_client.rb

```diff
@@ -39,8 +39,9 @@ def initialize(options = {})
       end
 
       @logger = options[:logger] || NullLogger.instance
+      disable_cache = options[:disable_cache] || ENV.fetch("DISABLE_JSON_API_CACHE", false)
 
-      if options[:disable_cache] || (options[:cache_size] == 0)
+      if disable_cache || options[:cache_size] == 0 # rubocop:disable Style/NumericPredicate
```

Comment:
D'oh bit annoying situation there. We should be ok using safe navigation operator but this Gem still supports 2.1. If I get a mo I'll switch it around to be testing against newer rubies and set minimum support of 2.3